content/blog/why-hoon.md

```diff
@@ -8,7 +8,7 @@ ship = "~rovnys-ricfer"
 image = "https://media.urbit.org/site/posts/essays/hoon.png"
 +++
 
-![](https://media.urbit.org/site/posts/essays/hoon.svg)
```

Comment:
Maybe there's a trick we can use to coerce this into desired width?

(Not telling you to find that out and fix it right now, but maybe it needs to become an issue? SVGs are great for many reasons, we should want to be able to use them.)

Reply:
The native svg was 480px. The png, exported at 2x, was 960px. The site uses a max-width: 100%, it’s just a small svg. Not a general svg bug.